docs/source/cli.rst

```diff
@@ -31,7 +31,7 @@ Read it
     optional arguments:
       -h, --help            show this help message and exit
       --pattern PATH, -p PATH
-                            paths to pattern definition files
+                            paths to pattern definition files or modules
```

Comment:
Isn't that kind of implied? "Your code failed to import my unimportable module" would get closed as wontfix